design-docs/done/build-initialisation.md

```diff
@@ -166,3 +166,14 @@ This story adds a commandline property `--gradle-version` to the `wrapper` task
 
 * Running `gradle wrapper --gradle-version 1.6` generates valid `wrapper.properties` with correct URL.
 
+# Story: User specifies target gradle distribution type when generating the wrapper
+
+This story adds a command line option called `--gradle-type` to the `wrapper` task to specify the desired Gradle distribution type:
```

Comment:
I'd be OK with that. I just thought it should start with `--gradle` to be consistent with `--gradle-version`. So, do you want me to use `--dist-type` instead`?